codegen/templates/tchannel_client.tmpl

```diff
@@ -102,7 +102,7 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		{{$serviceMethod := printf "%s::%s" $svc.Name .Name -}}
 		{{$methodName := (title (index $exposedMethods $serviceMethod)) -}}
 			{{if $methodName -}}
-			"{{$serviceMethod}}": "{{$methodName}}",
+			"{{$serviceMethod}}": "{{title .Name}}",
```

Comment:
This should not be changed, the `$methodName` is supposed to be the Go method name of the client.

Reply:
For the Edge Gateway usage of this template, there was always the assumption that the client and method names were a 1-1 mapping. Multiple services in one thrift disallow this. After talking offline with Chuntao offline, it sounds like we are going to just scrap the tchannel part of this diff and make consumers of metrics and logs switch to other metrics to avoid breaking changes in metrics and logging here.

Reply:
cc @abhishekparwal FYI

Reply:
> For the Edge Gateway usage of this template, there was always the assumption that the client and method names were a 1-1 mapping. Multiple services in one thrift disallow this. After talking offline with Chuntao offline, it sounds like we are going to just scrap the tchannel part of this diff and make consumers of metrics and logs switch to other metrics to avoid breaking changes in metrics and logging here.

To clarify, consumers of the metrics/logs will still look at the same metrics/logs but different tag/log field for thrift service and method. The client method name has always meant to be the Go method name of the client.